KMD.Identity.TestApplications.SAML.MVCCore/Config/ExtendedSaml2Configuration.cs

```diff
@@ -0,0 +1,12 @@
+﻿using ITfoxtec.Identity.Saml2;
+
+namespace KMD.Identity.TestApplications.SAML.MVCCore.Config
+{
+    public class ExtendedSaml2Configuration : Saml2Configuration
+    {
+        public string IdPMetadata { get; set; }
```

Comment:
What do you think about calling it something like IdPMetadataUrl?

Reply:
You are right, that would be better. Fixed.

Reply:
Thanks